waku-org/nwaku
# chore: Added metrics to liteprotocoltester

# Description
Added a dashboard to monitor testing performance from grafana dashboard

# Changes

- [X] Added similar metrics as in collected statistics
- [X] Fix negentropy lib reference in docker
- [X] Added prometheus and grafana into docker compose
- [X] Added liteprotocoltester dashboard

## How to test

Check liteprotocoltester readme. 
Run with waku-simulator as in the description
go to http://localhost:3033

![image](https://github.com/user-attachments/assets/b44bcf4d-32fc-4741-be1b-8710b03dc8cb)


## Issue
https://github.com/waku-org/nwaku/issues/2999
